_posts/0000-01-02-amribr0.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Step 7: Respond to a review
Your pull request is looking great!

Let’s add some content to your file. Replace line 5 of your file with a quotation or meme and witty caption. Remember: Markdown is supported.

⌨️ Activity: Change your file
Click the Files Changed tab in this pull request
Edit your newly added file
Replace line 5 with something new
Scroll to the bottom and click Commit Changes
Note : Can't find the button to edit the file? It may look like a pencil, or it may look like three dots.

Reply:
not Resolve it yet